### DISCUSSION TITLE
there are confusing JDocs code

### DISCUSSION THREAD

**Discussion by geraldsamosir** [webpack/webpack#18287] [Q&A]
there are some annoying documentation [here](https://github.com/webpack/webpack/blob/main/lib/json/JsonGenerator.js#L143)

```js
/**
	 * @param {NormalModule} module module for which the bailout reason should be determined
	 * @param {ConcatenationBailoutReasonContext} context context
	 * @returns {string | undefined} reason why this module can't be concatenated, undefined when it can be concatenated
	 */
	getConcatenationBailoutReason(module, context) {
		return undefined;
	}

```

at the method return just undefined why on documentation return string | undefined
that i know when to make sure on javascript condition can be like simillar if we dont use "==="  is only  undefined and null

**Comment by alexander-akait** [comment]
Sorry, I don't undestand you

**Reply by geraldsamosir** [reply]
I mean that code, not myself
